src/UI/Areas/Identity/Pages/Account/ResetPassword.cshtml.cs

```diff
@@ -4,11 +4,13 @@
 using System;
 using System.ComponentModel.DataAnnotations;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 
 namespace Microsoft.AspNetCore.Identity.UI.Pages.Account.Internal
 {
+    [AllowAnonymous]
```

Comment:
This seems incorrect as only a logged in user should be able to reset the password

Reply:
This should probably explicitly have Authorize attribute.

Reply:
We probably need a consistent experience of using these attributes. We currently do like the following to authorize some areas. @javiercn ?
https://github.com/aspnet/Identity/blob/release/2.1/src/UI/IdentityDefaultUIConfigureOptions.cs#L37-L38